square-frontend/src/components/Query.vue

```diff
@@ -19,16 +22,55 @@
           </div>
         </div>
       </div>
-      <div v-if="skillSettings.requiresContext" class="col-md-4 mt-4 mt-md-0">
+
+      <!-- Context and Answer Choices -->
+      <div class="col-md-4 me-auto mt-4 mt-md-0" v-if="skillSettings.skillType !== null">
         <div class="bg-light border border-warning rounded shadow h-100 p-3">
           <div class="w-100">
-            <label for="context" class="form-label">3. Provide context</label>
-            <textarea v-model="inputContext" class="form-control mb-2" style="resize: none; height: calc(38px * 7);"
-              id="context" :placeholder="contextPlaceholder" required />
-            <small class="text-muted">{{ contextHelp }}</small>
+            <!-- Context Input -->
+            <div class="row" v-if="skillSettings.requiresContext">
+              <label for="context" class="form-label">3. Provide context</label>
+              <textarea 
+                  v-if="skillSettings.skillType == 'multiple-choice'"
+                  v-model="inputContext"
+                  class="form-control form-control-lg mb-2"
```

Comment:
No, that didn't fix it. But the interface will change shortly, so I think it's not worth expending time on fixing this margin given that it may change soon.